docs/Troubleshooting.BuildingTheApp.md

```diff
@@ -13,7 +13,27 @@ For troubleshooting of other issue, refer to our [troubleshooting index](Trouble
 
 ## Android
 
-### Problem: minSdkVersion mistmatch
+### Problem: AAPT
```

Comment:
Lovin' this troubleshooting section. The heading, on the other hand, isn't very descriptive. `AAPT` can mean so many things...